MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -63,7 +63,7 @@
 import java.util.UUID;
 import java.util.concurrent.atomic.AtomicLong;
 
-public class MuxBaseExoPlayer extends EventBus implements IPlayerListener {
+public class  MuxBaseExoPlayer extends EventBus implements IPlayerListener {
```

Comment:
Please run the formatter before committing, this space is unnecessary.

Reply:
Sorry about that, fixed.